data/ui/ButtonRow.ui

```diff
@@ -9,10 +9,10 @@
       <object class="GtkBox">
         <property name="visible">True</property>
         <property name="can_focus">False</property>
-        <property name="margin_left">6</property>
-        <property name="margin_right">6</property>
-        <property name="margin_top">4</property>
-        <property name="margin_bottom">4</property>
```

Comment:
I'm not sure this is right, tbh. It should be the same layout as the list in the Keyboard control-center panel and it seems just those few pixels larger now (hard to tell though). what's the specific reason for this?